jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/sam/SamTemplateUtils.kt

```diff
@@ -83,6 +82,22 @@ object SamTemplateUtils {
         version = codeUri.get("Version").textValue()
     )
 
+    fun getCodeLocation(template: Path, logicalId: String): String = readTemplate(template) {
```

Comment:
nit: Add doc link?

Reply:
I am not aware of a good candidate, since the sam template specification is the built template, not the pre-built form.

Added:

```
    /**
     * Returns the location of the Lambda source code as per SAM build requirements
     */
```